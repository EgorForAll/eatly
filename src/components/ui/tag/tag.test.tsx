import {screen, render} from "@testing-library/react";
import Tag from "./tag";
import '@testing-library/jest-dom'

const tagMock: string = 'history';
const tagsMock: string[] = ['history', 'crime', 'american']

test('Tag renders correctly', () => {
    render(<Tag tag={tagMock} tags={tagsMock}/>)
    const tag = screen.getByTestId('tag');
    expect(tag).toMatchSnapshot();
})