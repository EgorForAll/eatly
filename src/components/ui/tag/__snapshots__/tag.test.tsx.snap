// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag renders correctly 1`] = `
<span
  class="tag"
  data-testid="tag"
>
  #
  history
  ,
</span>
`;
